subcommand/server-acl-init/command_ent_test.go

```diff
@@ -1,3 +1,5 @@
+// +build enterprise
```

Comment:
@kschoche without this the enterprise tests run with the oss binary (and they'll all fail)

Reply:
Ahhh completely forgot to switch this back, that explains so much

Reply:
![image](https://user-images.githubusercontent.com/1034429/97648057-0d030500-1a11-11eb-9bd6-d4f2896d9e4e.png)

I set this in Goland to get everything to build (add the enterprise tag).